content/verification/veteran-verification/release-notes/2023-05-18

```diff
@@ -0,0 +1,26 @@
+We added a new version (v2) of the Veteran Verification API to the developer portal, and the API has been renamed to the 'Veteran Service History and Eligibility API'.
+
+The following enhancements and new features are available in v2:  
+
+All of the API's endpoints can now be accessed using either Authorization Code Flow OAuth or Client Credentials Grant (CCG) OAuth.   
+There are two versions of each endpoint, a GET when ICN is provided and a POST when demographic details will be used to match a Veteran.   
+The disability_rating endpoint now includes the diagnostic code and name associated with each individual rating.
+The service_history endpoint now includes additional information:
+A service episode's component_of_service identifies unique sub-categories within a main branch of service, such as "Reserves."  In the prior version (Veteran Verification, v1), the branch_of_service is consistently a main branch (e.g., Army, Navy...) with no sub-category values such as "Reserves."  Adding this distinct field for this purpose clarifies which branch of service a period of Reserve duty is associated with.  
```

Comment:
Incorporated this sentence you mentioned: 'This will reveal when a Veteran was in the Reserves for their branch of military service.'

```suggestion
A service episode's component_of_service identifies unique sub-categories within a main branch of service, such as "Reserves."  In the prior version (Veteran Verification, v1), the branch_of_service is consistently a main branch (e.g., Army, Navy...) with no sub-category values such as "Reserves."  Adding this distinct field reveals when a Veteran was in the Reserves for their branch of military service.  
```